packages/api-reference/src/components/Section/CompactSection.vue

```diff
@@ -39,10 +39,9 @@ watch(
       :class="{ 'collapsible-section-trigger-open': open }"
       type="button"
       @click="open = !open">
-      <ScalarIcon
-        :icon="open ? 'ChevronDown' : 'ChevronRight'"
-        size="md"
-        thickness="1.5" />
+      <ScalarIconCaretRight
+        class="size-4"
```

Comment:
You can throw in 
```suggestion
        class="size-4 transition-transform"
```

 So it animates like the other carets too if you want 😊